easybuild/framework/easyconfig/types.py

```diff
@@ -83,9 +95,7 @@ def is_value_of_type(value, typ_spec):
                     type_ok &= check_ok
                     _log.debug("Check for %s requirement (%s) %s for %s", er_key, extra_reqs[er_key], msg, value)
                 else:
-                    raise EasyBuildError("Unknown type requirement specified: %s", er_key)
-        else:
-            raise EasyBuildError("Don't know how to check value with parent type %s", parent_type)
+                   raise EasyBuildError("Unknown type requirement specified: %s", er_key)
```

Comment:
what happened with the indent here? not a multiple of 4-space indent anymore (or maybe it wasn't before)?